index.html

```diff
@@ -5,31 +5,32 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
-    <title>Odin-Project-Landing-Page</title>
+<title>Odin-Project-Landing-Page</title><!---Don't forget to change the title. This is what pops up in the browser tab-->
```

Comment:
These are how I added the feedback. I wanted to make sure everything showed up the right way, so I put them in comment blocks. It should all show up as green on this side of the code review tab